### PR TITLE
pcustom: Don't barf if struct dir doesn't exist

### LINKED ISSUE #69: FileNotFoundError with `dt -l` when the path doesn't exist
Easy to fix, i'll do it.

```
gef➤  dt -l
[+] Listing custom structures:
Python Exception <class 'FileNotFoundError'> [Errno 2] No such file or directory: '/tmp/user/1000/gef/structs':
Error occurred in Python command: [Errno 2] No such file or directory: '/tmp/user/1000/gef/structs'
```

### PR DESCRIPTION
**Pull request by Grazfather**
Fixes #69

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -2620,9 +2620,14 @@ def create_or_edit_structure(self, structure_name):
...
-            if not filen.endswith("py"): continue
-            print("%s %s" % (right_arrow, filen.replace(".py", "")))
+        path = self.get_setting("struct_path")
+        try:
+            for filen in os.listdir(path):
+                name, ext = os.path.splitext(filen)
+                if ext != ".py": continue
+                print("%s %s" % (right_arrow, name))
+        except OSError:
+            print("Cannot open %s. Create struct directory or set pcustom.struct_path" % path)
```
Comment (hugsy) [inline gef.py]: Use `err()` or `warn()` instead

File: gef.py [hunk 2]
```diff
@@ -2620,9 +2620,14 @@ def create_or_edit_structure(self, structure_name):
...
         info("Listing custom structures:")
-        for filen in os.listdir(self.get_setting("struct_path")):
-            if not filen.endswith("py"): continue
-            print("%s %s" % (right_arrow, filen.replace(".py", "")))
+        path = self.get_setting("struct_path")
+        try:
+            for filen in os.listdir(path):
+                name, ext = os.path.splitext(filen)
+                if ext != ".py": continue
+                print("%s %s" % (right_arrow, name))
```
Comment (hugsy) [inline gef.py]: Use `ok()` instead of `print()`

**Comment by Grazfather** [comment]
Force pushed to fix.

**Comment by hugsy** [comment]
:+1:

**Pull request merged**